handel/src/services/s3/index.ts

```diff
@@ -144,20 +144,15 @@ function getS3EventFilters(filterList: S3ServiceEventFilterList | undefined): AW
  */
 
 export function check(serviceContext: ServiceContext<S3ServiceConfig>, dependenciesServiceContexts: Array<ServiceContext<ServiceConfig>>): string[] {
-    const errors = [];
+    const errors: string[] = checkPhase.checkJsonSchema(`${__dirname}/params-schema.json`, serviceContext);
+
     const validAcls = ['AuthenticatedRead', 'AwsExecRead', 'BucketOwnerRead', 'BucketOwnerFullControl', 'LogDeliveryWrite', 'Private', 'PublicRead'];
```

Comment:
This line and the one after it can be deleted because they're not referenced anywhere anymore after deleting those two if statements below.